tests/make_test_data.py

```diff
@@ -60,3 +62,60 @@
     'celldb.azimuth_degrees': degrees})
 
 df.to_csv('testdata.csv', index=False)
+
+# extended testdata - 3 days with random time increments
+number_of_days = 3
+id = 0
+ids = []
+track = []
+sensor = []
+timestamp = []
+lon = []
+lat = []
+degrees = []
+# track 1
+for i in range(number_of_days*1440):
+    ids.append(id)
+    id += 1
+    track.append("TelA")
+    sensor.append("A1")
+    timestamp.append((startdate + timedelta(
+        minutes=i+0.5*(random.random()-.5))).isoformat(" "))
+    lat.append(round(startloc[0] + (i*0.001), 4))
+    lon.append(round(startloc[1] + (i*0.001), 4))
+    degrees.append(0)
+
+# track 2
+for i in range(number_of_days*1440):
+    ids.append(id)
+    id += 1
+    track.append("TelA")
+    sensor.append("A2")
+    timestamp.append((startdate + timedelta(
+        minutes=i+0.5*(random.random()-.5))).isoformat(" "))
+    lat.append(round(startloc[0] + (i*0.001) + 0.001*random.random(), 4))
+    lon.append(round(startloc[1] + (i*0.001) + 0.001*random.random(), 4))
+    degrees.append(0)
```

Comment:
Afhankelijk van het doel van deze tracks wil je misschien dat de track naar het noord-oosten gaat met om de zoveel meter een punt. Dat doet deze code niet. Zie https://stackoverflow.com/questions/24427828/calculate-point-based-on-distance-and-direction